docs/sources/api/_index.md

```diff
@@ -209,37 +217,37 @@ $ curl -G -s  "http://localhost:3100/loki/api/v1/query" --data-urlencode 'query=
   }
 }
 ```
+Set the `x-scope-orgid` header,
+as defined in [Grafana Loki Multi-Tenancy](../operations/multi-tenancy/),
+to query specific tenants.
+Here is the same example query applied to the single tenant called `Tenant1`:
 
 ```bash
-$ curl -G -s  "http://localhost:3100/loki/api/v1/query" --data-urlencode 'query={job="varlogs"}' | jq
-{
-  "status": "success",
-  "data": {
-    "resultType": "streams",
-    "result": [
-      {
-        "stream": {
-          "filename": "/var/log/myproject.log",
-          "job": "varlogs",
-          "level": "info"
-        },
-        "values": [
-          [
-            "1568234281726420425",
-            "foo"
-          ],
-          [
-            "1568234269716526880",
-            "bar"
-          ]
-        ],
-      }
-    ],
-    "stats": {
-      ...
-    }
-  }
-}
+curl -H 'x-scope-orgid:Tenant1' \
+  -G -s "http://localhost:3100/loki/api/v1/query" \
+  --data-urlencode \
+  'query=sum(rate({job="varlogs"}[10m])) by (level)' | jq
+```
+
+To query against the three tenants `Tenant1`, `Tenant2`, and `Tenant3`,
```

Comment:
```suggestion
If you have cross tenant query federation enabled (link to docs), you can query multiple tenants at once. 

To query against the three tenants `Tenant1`, `Tenant2`, and `Tenant3`,
```

Reply:
The sentence above this one contains the link to the discussion on enabling this, so I don't want to repeat the prose or the link.